package.json

```diff
@@ -13,6 +13,7 @@
     "prepare": "husky"
   },
   "dependencies": {
+    "@internationalized/date": "^3.6.0",
```

Comment:
We have date-fns, use this instead of providing another packages. Also I think you don't use it or am I wrong. If it is for the localized calendar, I would not use it, since everything is in english, so calendar should be in english as well!